app/views/articles/_sidebar_campaign.html.erb

```diff
@@ -1,16 +1,19 @@
 <div class="widget">
+  <% if SiteConfig.campaign_sidebar_image %>
+    <img src="<%= SiteConfig.campaign_sidebar_image %>" />
```

Comment:
I suppose it's not the best idea to put an image like this, and there is a problem that the image is not displaying properly right now. I need @Ridhwana 's advice on this.